README.md

```diff
@@ -30,165 +32,458 @@ The two guiding principles are:
+```css
+.headline {
+  color: red;
+}
+```
 
-**Therefore we have bundled them together in the convenience preset [`hops-preset-defaults`](https://github.com/xing/hops/tree/master/packages/preset-defaults)**.
+And when we now import this file we will have access to all its class names which we can use to style our elements accordingly:
 
-Together these core presets expose the following CLI commands:
+**`src/app.js`**
 
-- `hops build [-p, --production] [-s, --static]` - execute a build and write assets to filesystem.
-  - `--production` is an alias for `NODE_ENV=production` and will create a production build (minifies the bundle, externalizes the source maps, etc).
-  - `--static` will render the configured `locations` to static HTML files.
-- `hops develop` - start a development server with hot module reloading enabled.
-- `hops serve` - start an Express.js server for your application (don't forget to execute `hops build` before).
-- `hops start [-p, --production] [-s, --static]` - either starts a development server or executes a production build and serves that.
-  - `--production` is an alias for `NODE_ENV=production` and will create a production build and afterwards start a production ready Express.js server.
-  - `--static` is only relevant when executed together with the `--production` argument and will do a static build instead (renders static HTML pages) and serve them afterwards.
+```javascript
+import React from 'react';
+import { render } from 'hops';
+import styles from './styles.css';
+
+export default render(<h1 className={styles.headline}>Hello World!</h1>);
+```
 
-We also recommend that you always install the [`hops-react`](https://github.com/xing/hops/tree/master/packages/react) preset in order to start using [React](https://reactjs.org/) in your application.
+### Code-splitting and `hops`s `Import()` function
 
-### Installing / configuring a preset
+You can use code-splitting (or bundle-splitting) to reduce the size of the assets that your users have to download by creating multiple bundles that contain only the code that is actually needed. Under the hood this uses [dynamic imports]() which webpack will transform into [separate bundles](). Hops provides support for this with a custom component (called `Import()`) in order to render the actual content on the server-side and render a placeholder during client-side navigation.
```

Comment:
Shall we link to the webpack [code-spitting](https://webpack.js.org/guides/code-splitting/) docs?

Reply:
done